toolshed/routes/ai/llm/generateText.test.ts

```diff
@@ -0,0 +1,201 @@
+import { assertEquals } from "@std/assert";
+import env from "@/env.ts";
+
+// Import the functions for testing
+import { cleanJsonResponse, configureJsonMode } from "./generateText.ts";
+
+if (env.ENV !== "test") {
+  throw new Error("ENV must be 'test'");
+}
+
+Deno.test("JSON mode configuration", async (t) => {
+  // Test for Groq models
+  await t.step("Configures JSON mode correctly for Groq models", () => {
```

Comment:
The `Deno.test`/`t.step` interface was used in basic-flow to handle some integration/astral slowness/assertions during development, but could probably be ported back to the BDD style -- if we don't have an explicit reason to use the inner `t.step` interface, consider the BDD wrapper around stepped tests e.g. `import { describe, it, beforeEach } from "@std/testing/bdd"`